panimg/image_builders/tiff.py

```diff
@@ -328,6 +332,28 @@ def _convert_to_tiff(
     return new_file_name
 
 
+def _convert_dicom_wsi_dir(
+    gc_file: GrandChallengeTiffFile,
+    file: Path,
+    output_directory: Path,
+    file_errors: Dict[Path, List[str]],
+):
+    wsidicom_dir = file.parent
+    try:
+        new_file_name = output_directory / file.name / f"{gc_file.pk}.tif"
+        new_file_name.parent.mkdir()
+
+        wsi_dcm_to_tiff(wsidicom_dir, new_file_name)
+    except Exception as e:
+        file_errors[file].append(str(e))
+    else:
+        gc_file.path = new_file_name
+        gc_file.associated_files = [
```

Comment:
What happens when a folder contains files from more than 1 dicom? Will the import or conversion to tiff fail?

Reply:
No, it will convert all .dcm files in a directory into a single tiff file.

Reply:
Voor het gebruik van de testimages van Nema kunnen we deze downloaden, zoals wsidicom ook doet, zie:
https://github.com/imi-bigpicture/wsidicom/blob/main/tests/download_test_images.py

Reply:
Those files are the ones I used locally, the python script grabs them from the nema FTP server. Do we need to put any license or disclaimer somewhere before I include them?

Reply:
As long as we download the files from their original location during testing, we don't need to add any disclaimers or licenses. That's (apparently) why wsidicom, and also pydicom, construct their tests that way. Just don't include the files themselves, and it should be good.

Reply:
Then I'm actually not so sure we should add them. Currently, the tests run in under 10 seconds. Downloading the data takes >10 minutes on my machine. Even if we skip the large files (>10mb, would've left these out anyway to not bloat the repo) it takes >30 seconds. So it will significantly slow down the tests, I don't think that is worth it.

Reply:
I agree, that's not worth it.